developer-docs-site/docs/nodes/ait/steps-in-ait3.md

```diff
@@ -8,56 +8,93 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # Steps in AIT-3
 
-:::caution DRAFT-only
-These AIT-3 docs are draft-only for now.
-:::
+See below the summary flowcharts and detailed steps you will execute while participating in AIT-3.
 
-See below the summary flowcharts and detailed steps you will execute while participating.
+:::caution Chrome browser only
+The new Petra (Aptos Wallet) is supported only on the Chrome browser. Hence, for all the below AIT-3 tasks, make sure that you use only the Chrome browser. 
+:::
 
-## Install wallet, deploy the validator node and register 
+## Sign-in, connect Wallet and complete survey
 
 Participants in the AIT-3 program must demonstrate the ability to configure and deploy a node, and pass the minimum performance requirements as reported by the [Node Health Checker](/nodes/node-health-checker). 
 
 ### Summary steps
 
+<center>
 <ThemedImage
 alt="Signed Transaction Flow"
 sources={{
-    light: useBaseUrl('/img/docs/install-node-and-register.svg'),
-    dark: useBaseUrl('/img/docs/install-node-and-register.svg'),
+    light: useBaseUrl('/img/docs/sign-in-to-survey.svg'),
+    dark: useBaseUrl('/img/docs/sign-in-to-survey.svg'),
   }}
 />
+</center>
 
 ### Detailed steps
 
+
 To participate in the AIT-3 program, follow the below steps. Use these steps as a checklist to keep track of your progress. Click on the links in each step for a detailed documentation.
 
-1. Navigate to the [Aptos Community page](https://aptoslabs.com/community) to register or sign in to your account.
+1. Navigate to the [Aptos Community page](https://aptoslabs.com/community) and follow the steps, starting with registering or signing in to your Discord account.
+
+2. Before you click on Step 2 **CONNECT WALLET**: 
+   1. Install the Petra (Aptos Wallet) extension using Step 3 instructions, and 
+   2. Create the first wallet using Step 4 instructions.
+3. **Install** the Petra (Aptos Wallet) extension on your Chrome browser by [following the instructions here](/guides/building-wallet-extension).
+    
+4. <span id="create-wallet">Create the first wallet using Petra (Aptos Wallet)</span>. 
+
+  **This first wallet will always be the owner wallet**. 
 
-2. Install Petra (Aptos Wallet) follow the instruction [here](/guides/building-wallet-extension).
+   1. Open the Aptos Wallet extension from the Extensions section of the Chrome browser, or by clicking on the puzzle piece on top right of the browser and selecting Aptos Wallet.
+   2. Click **Create a new wallet**. 
+   3. When you are done creating the wallet, go to **Extensions** > **Aptos Wallet** and click on the gear icon on the bottom right. You will see the **Settings** screen. Click on the **Network** button and make sure the AIT-3 network is selected. 
+   4. Next, click again on the gear icon on the bottom right to go to the **Settings** page. Click on the **Credentials**. Copy and paste and save both the **Private key** and **Address** keys in a separate text file for later use. **This private key is the owner key**.
```

Comment:
remove step 4, we don't want people to export their keys unless absolutely needed